invenio_rdm_records/oaiserver/services/services.py

```diff
@@ -130,7 +131,13 @@ def search(self, identity, params):
 
         search_params = self._get_search_params(params)
 
-        oai_sets = OAISet.query.order_by(
+        query_param = search_params["q"]
+        filters = []
+
+        if query_param:
+            filters.extend([OAISet.name.ilike(f"%{query_param}%"), OAISet.spec.ilike(f"%{query_param}%")])
```

Comment:
For now it's filtering based on `name` or `spec`. Do we want to have more fields? 
Should this be configurable?

Reply:
I think we could merge and check if we receive feedback on that. Maybe we should include a tooltip for the searchbox informing about this limitation ?